core/amber/src/main/python/core/models/operator.py

```diff
@@ -45,7 +46,7 @@ def output_schema(self) -> Schema:
     @output_schema.setter
     @overrides.final
     def output_schema(
-        self, raw_output_schema: Union[Schema, Mapping[str, str]]
+            self, raw_output_schema: Union[Schema, Mapping[str, str]]
```

Comment:
please avoid unnecessary changes. to fix format, run `black .` under `core/amber/src/main/python`

Reply:
fix